src/main/resources/l10n/Menu_de.properties

```diff
@@ -1,28 +1,29 @@
+#X-Generator: crowdin.com
 Abbreviate\ journal\ names\ (ISO)=Zeitschriftentitel abkürzen (&ISO)
 Abbreviate\ journal\ names\ (MEDLINE)=Zeitschriftentitel abkürzen (&MEDLINE)
 About\ JabRef=Über &JabRef
 Append\ library=Bibliothek &anhängen
 Autogenerate\ BibTeX\ keys=&BibTeX-Keys automatisch generieren
 Close\ library=Bibliothek s&chließen
-Copy=&Kopieren
```

Comment:
Seems like we loose our mnemonic keys!

Reply:
I try to work on it. Therefore, you see these pull requests. Please ignore them until I say, it is OK!

Reply:
The issue is that I loaded some wrong translations and they are kept somewhere at Crowdin.

Reply:
Note that we currently also loose the German translation. So please be patient!